### PR TITLE
Convert devs.json to single JSON files

### PR DESCRIPTION
**Pull request by jromest**
**Description**
This PR converts devs.json to single JSON files, this way we can avoid conflicts to contributors editing the same file.

**Summary of Changes**
- add `gatsby-transformer-json`
- add devs JSON in Dev Board page via graphql
- update pre-push git hook and add `yarn lint`
- update README

### CODE REVIEW

**Review by franreysaycon — APPROVED** [review]
Minor nit. maybe you can make the file forEach to be more parallel instead of one by one. this is theoretically though. something along the lines of Promise.all.

File: README.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ More for information about how you can contribute to this project, check our [co
 
 ## 📋 Enlisting in the Dev Board
 
-1. Edit the [`content/devs.json`](https://github.com/reactph/reactjsph-website/blob/master/content/devs.json) file.
+1. Add a JSON file in the [`content/devs`](https://github.com/reactph/reactjsph-website/blob/master/content/devs) folder with `firstname-lastname.json` as filename, all lowercase, separated by `-` (e.g. `juan-dela-cruz.json`).
```
Comment (galacemiguel) [inline README.md:59]: **`NITPICK`**

```suggestion
1. Add a JSON file to the [`content/devs`](https://github.com/reactph/reactjsph-website/blob/master/content/devs) folder with filename `firstname-lastname.json` (all lowercase, separated by `-`), e.g., `juan-dela-cruz.json`.
```

File: README.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ More for information about how you can contribute to this project, check our [co
...
 
-2. At the end of the array, add an object describing yourself given the format below:
+2. Create an object describing yourself given the format below, here's an [example](https://github.com/reactph/reactjsph-website/blob/master/content/devs/franrey-anthony-saycon.json).
```
Comment (galacemiguel) [inline README.md:61]: **`NITPICK`**

```suggestion
2. Within that file, define an object describing yourself given the format below. Here's an [example](https://github.com/reactph/reactjsph-website/blob/master/content/devs/franrey-anthony-saycon.json).
```

File: README.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ More for information about how you can contribute to this project, check our [co
   | `skills` | String array of your skills/technologies that you would like to promote | <ul><li>Only the first 5 will be listed, but feel free to add as many as you like (We plan to allow devs to be filtered by skills in the future)</li></ul> |
   | `contacts` | Object array of contact information where each item has a `type` and a `url` | <ul><li>`type` can be any of the following: "behance", "email", "github", "linkedin", "twitter", "website"</li><li>`url` must be a URL corresponding to the `type` selected</li></ul> |
 
-3. Open a pull request with the title `[DEV BOARD] {YOUR_NAME}` using the template below:
+3. Commit your changes with the title `[DEV BOARD] {YOUR_NAME}`, open a pull request with the same title, and use the template below:
```
Comment (galacemiguel) [inline README.md:73]:
```suggestion
3. Commit your changes with the message `[DEV BOARD] {YOUR_NAME}` and open a pull request with the same title, following the template below:
```

**Review by galacemiguel — APPROVED** [review]
Good call, @jromest! Just some grammar nitpicks, but will approve already. 👍

**Review by galacemiguel — APPROVED** [review]

**Pull request merged**